lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -220,7 +220,7 @@ class Parser
     /**
      * The custom last tree walker, if any, that is responsible for producing the output.
      *
-     * @var class-string<TreeWalker>
+     * @var class-string<SqlWalker>|null
```

Comment:
As explained on #9551, output walkers cannot be arbitrary implementations of `TreeWalker`, we do expect an `SqlWalker` here.